integr8ly/integreatly-operator
# MGDAPI - 2351 - Limitador

# Description

Migration to use Limitador as rate limiting solution

Jira:
* https://issues.redhat.com/browse/MGDAPI-2351

## Verification
* Checkout master
* Install RHOAM from master
* Checkout this branch
* Run from this branch
* Ensure installation completes successfully and rate limiting solution is migrated 
  * Rate limit deployment in `redhat-rhoam-marin3r` is using limitador image
  * StatsD deployment is deleted in `redhat-rhoam-marin3r`
  * StatsD service is deleted in `redhat-rhoam-marin3r`
* Sign into 3scale and complete tutorial api 
* Curl the tutorial endpoint to ensure rate limiting is still functional
```
# Rate limiting should be at 70 requests per minute for 100K option
i=0

while [ $i -le 75 ] 
do 
  curl "<3scale_api>" -v  
  echo Number: $i
  ((i++))
done
```
* Deploy workload web app
  * https://github.com/integr8ly/workload-web-app#rhoam-clusters
* Check the customer grafana rate limiting dashboard and ensure the dashboard are still functional
* Ensure changes of quota still works as intended
```
TEST="A34" make test/e2e/single 
```
## Type of change




## Checklist
